task/06-conditions-n-loops-tasks.js

```diff
@@ -63,10 +73,11 @@ function getFactorial(n) {
  *   -1,1  =>  0  ( = -1 + 0 + 1 )
  */
 function getSumBetweenNumbers(n1, n2) {
-    throw new Error('Not implemented');
+  let result = 0;
+  while (n2 != n1 - 1) result += n2--;
```

Comment:
do not modify arguments if not required. Do not do assignements in the loop definition